README.md

```diff
@@ -86,7 +86,8 @@ from fastapi import FastAPI
 from fastapi.responses import HTMLResponse
 from openapipages import Elements, RapiDoc, ReDoc, Scalar, SwaggerUI
 
-app = FastAPI()
+# Disable the built-in /redoc page so we can make a custom one.
+app = FastAPI(redoc_url=None)
```

Comment:
We can alter the URL path for the ReDoc page instead of overriding the path of the built-in documentation, WDYT?

Reply:
We could, but I think overriding the built-in one is more useful. I doubt anyone wants two very similar but subtly different ReDoc pages in their service, and `/redoc` is the obvious name for the ReDoc page. We'll also need comments to explain why we are not using the `/redoc` URL and what to do if one wants to use it, so it will just make everything more complicated.

Reply:
> We could, but I think overriding the built-in one is more useful. I doubt anyone wants two very similar but subtly different ReDoc pages in their service. Imagine the confusion this can lead to 😄 And `/redoc` is the obvious name for the ReDoc page.

My only concern was the ReDoc version, but as far as I know, FastAPI will stop using ReDoc in the coming period. So I don't see a problem anymore.

> We'll also need comments to explain why we are not using the `/redoc` URL and what to do if one wants to use it, so it will just make everything more complicated.

Thats fair.